scripts/locust/service.py

```diff
@@ -29,127 +54,158 @@ def trim_url(url: str):
-                                                    '%22%3A%7B%22is%22%3A%5B%22temporal%20lobe%22%5D%7D%7D%7D'))
-                group.spawn(lambda: self.client.get('/repository/files?filters=%7B%22file%22%3A%7B%22organPart'
-                                                    '%22%3A%7B%22is%22%3A%5B%22temporal%20lobe%22%5D%7D%7D%7D'
-                                                    '&size=15&sort=sampleId&order=desc'))
-
-        @seq_task(2)
-        @task(1)
-        def download_manifest(self):
-            self.client.get('/repository/summary?filters=%7B%22file%22%3A%7B%22organPart%22%3A%7B%22'
-                            'is%22%3A%5B%22temporal%20lobe%22%5D%7D%7D%7D')
-            export_url = ('/manifest/files?filters=%7B%22file%22%3A%7B%22organPart%22%3A%7B%22is%22%3A%5B%22'
-                          'temporal%20lobe%22%5D%7D%2C%22fileFormat%22%3A%7B%22is%22%3A%5B%22fastq.gz%22%2C%22'
-                          'bai%22%2C%22bam%22%2C%22csv%22%2C%22results%22%2C%22txt%22%5D%7D%7D%7D&format=bdbag')
-            with self.client.get(export_url, catch_response=True, allow_redirects=False) as response:
-                # this is necessary because non 2xx response are counted as failures unless specified like this
-                if response.status_code == 301 or (200 <= response.status_code < 300):
+        browse_page(self.client, 'projects', {"fileFormat": {"is": ["mtx"]}}, **browser_search_params)
+
+
+class ManifestTaskSet(SequentialTaskSet):
+    """
+    Filter files by organ part and download a BDBag-format manifest.
+    """
+
+    # Select islet of Langerhans since it's present in the develop deployment.
+    organ_part_filter = {"organPart": {"is": ["islet of Langerhans"]}}
+    manifest_file_format_filter = {"fileFormat": {"is": ["fastq.gz", "bai", "bam", "csv", "results", "txt"]}}
+
+    @task
+    def start_page(self):
+        browse_page(self.client, 'files', {}, size=15)
+
+    @task
+    def filter_organ_part(self):
+        browse_page(self.client,
+                    'files',
+                    self.organ_part_filter,
+                    **browser_search_params)
+
+    @task
+    def download_manifest(self):
+        self.client.get(endpoint('/repository/summary', filters=json.dumps(self.organ_part_filter)),
+                        name='/repository/summary')
+        export_url = endpoint('/manifest/files',
+                              filters=json.dumps({**self.organ_part_filter,
+                                                  **self.manifest_file_format_filter}),
+                              format='bdbag')
+        with self.client.get(export_url,
+                             name='/manifest/files',
+                             catch_response=True,
+                             allow_redirects=False) as response:
+            # this is necessary because non 2xx response are counted as failures unless specified like this
```

Comment:
80

Reply:
```suggestion
            # This is necessary because non 2xx response are counted as failures unless specified like this
```

Reply:
What do you mean by "80"?

Reply:
I am soooo tired of saying the same thing a million times.

I mean wrap on 80 as specified in C guide.